tests/unit/test_help.py

```diff
@@ -9,3 +9,16 @@ def test_help():
         stderr=subprocess.STDOUT, env=os.environ.copy(),
     )
     assert output
+
+
+def test_count_of_description_pre_option():
+    test_command = 'pipenv install --help'
+    test_line = '--pre Allow pre-releases.'
+    out = subprocess.Popen(test_command.split(), stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
This case should be in `integration` directory and use `PipenvInstance` to keep consistency.